### PR TITLE
Reconfigures kba asset loading to occur asynchronously, adds skeleton

### PR DESCRIPTION
**Pull request by AllenBW**
fixes https://projects.engineering.redhat.com/browse/RHCLOUD-1612

### what it looks like!
LOADING

### CODE REVIEW

**Comment by AllenBW** [comment]
cc @robcwilliams @katierik

**Comment by AllenBW** [comment]
ooo also @dkuc

**Comment by AllenBW** [comment]
travis is cray, those aren't failures on changed code

File: packages/inventory-insights/src/Insights.js [hunk 1]
```diff
@@ -93,6 +90,20 @@ class InventoryRuleList extends Component {
         }
     }
 
+
```
Comment (karelhala) [inline packages/inventory-insights/src/Insights.js]: Remove this emtpy line and we are good to go!

**Pull request merged**